### PR TITLE
Elice Greenberg- Images

### PR DESCRIPTION
**Pull request by elicegreen**

### CODE REVIEW

**Comment by elicegreen** [comment]
Can't figure out why the code only changes one image

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -7,8 +7,36 @@
...
+#Calculate the new width and height to resize to.
+    if width > height:
+        height = int((SQUARE_FIT_SIZE / width) * height)
+        width = SQUARE_FIT_SIZE
+    else:
+        width = int((SQUARE_FIT_SIZE / height) * width)
+        height = SQUARE_FIT_SIZE
+                    
+#Resizes the image.
+print('Resizing %s...' % (filename))
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @elicegreen this code is not indented which means it only applies outside of the loop

**Comment by elicegreen** [comment]
Thank you!! Got it now!

File: resizeAndAddLogo.py (hunk 1 above)
Comment (elicegreen) [inline resizeAndAddLogo.py]: I thought I had it... I indented (got everything into the 'for' loop) and it created all the files.... but it's not passing the check now :-/

**Comment by halfaro1** [comment]
@elicegreen 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**